apm-agent-core/src/test/resources/log4j2-test.xml

```diff
@@ -15,7 +15,5 @@
         <Logger name="com.github.dockerjava" level="info"/>
         <Logger name="org.testcontainers" level="info"/>
 
-        <!-- agent always in debug for easier -->
-        <Logger name="co.elastic.apm" level="debug"/>
```

Comment:
[for reviewer] this one is sneaky as it was setting the log level directly at the logging implementation level, hence bypassing any configuration we can make with the `log_level` configuration option. Removing it from here allows to let the one from `TestLogLevel` be used as expected.